electrumsv/wallet.py

```diff
@@ -963,6 +971,13 @@ def get_address_history(self, address: Address):
 
     async def set_address_history(self, script_hash: str, address: Address,
             hist: Dict[str, List[Tuple[str, int]]], tx_fees: Dict[str, int]):
+        # We got here due to incoming transactions for subscriptions to new / "observed addresses"
+        # If we prepared this tx locally then we will already have a TxCacheEntry which
+        # HasByteData flag set. (StateSigned | StateDispatched | StateReceived)
+        # In such cases we can fast track --> StateCleared
```

Comment:
Get rid of all the `-->` and use plain language. All comments wrapped at 120 characters.